packages/sdk/src/contracts/interfaces.contract.ts

```diff
@@ -11,11 +11,11 @@ import {Holograph} from './index'
 type HolographInterfacesFunctionNames = ExtractAbiFunctionNames<typeof HolographInterfacesABI>
 
 export enum ChainIdType {
-  UNDEFINED, //  0
-  EVM, //        1
-  HOLOGRAPH, //  2
-  LAYERZERO, //  3
-  HYPERLANE, //   4
+  UNDEFINED, // 0
```

Comment:
I think we should have a comment here that links to the solidity code: https://github.com/holographxyz/holograph-protocol/blob/develop/contracts/enum/ChainIdType.sol